wallet/src/plan/planner.rs

```diff
@@ -77,8 +76,8 @@ impl<R: RngCore + CryptoRng> Planner<R> {
 
     /// Set a memo for this transaction plan.
     #[instrument(skip(self))]
-    pub fn memo(&mut self, memo: MemoPlaintext) -> &mut Self {
-        self.plan.memo_plan = Some(MemoPlan::new(&mut self.rng, memo));
+    pub fn memo(&mut self, memo: String) -> &mut Self {
+        self.plan.memo_plan = Some(MemoPlan::new(&mut self.rng, memo).unwrap());
```

Comment:
I don't think we want to `unwrap()` here, because then we'll panic if an over-length memo is ever supplied. Instead we should return a `Result`, so that we just error out if the memo is too long.

Reply:
Fixed in https://github.com/penumbra-zone/penumbra/pull/1548/commits/1da961ff0b048af60f4f8320794490199fbdf209 rather than waiting for another review round-trip, since this looks great otherwise